mne/source_space.py

```diff
@@ -1426,12 +1427,17 @@ def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
     """
     subjects_dir = get_subjects_dir(subjects_dir)
 
+    if overwrite is not None:
+        raise ValueError("Parameter 'overwrite' is deprecated. Use "
+                         "mne.write_source_spaces instead.")
```

Comment:
We should do these deprecations in `setup_source_space` and `make_forward_solution`, too